src/reducers/searchReducer/searchReducer.test.js

```diff
@@ -0,0 +1,22 @@
+import { searchReducer } from './searchReducer';
```

Comment:
I don't think we need this reducer anymore since I refactored the search.

Reply:
ok, removed